source/reference/bson-types.txt

```diff
@@ -37,8 +37,8 @@ ObjectId
 --------
 
 ObjectIds are small, likely unique, fast to generate, and ordered.
-ObjectId values consists of 12-bytes, where the first four bytes are a
-timestamp that reflect the ObjectId's creation, specifically:
+ObjectId values consist of 12-bytes, where the first four bytes are a
```

Comment:
@kay-kim: I'm not sure if "12-bytes" is correct here. In context, "12 bytes" may be more appropriate, as we're not using "12-bytes" as an adjective.